### ISSUE TITLE
ECEF to ECI conversion

### ISSUE THREAD

**Issue by Tarlan0001** [JuliaGeo/Geodesy.jl#41]
Is there some kind of ECEF to ECI conversion (from fixed to inertial)? I didn't find in the documentation and was a little surprised as it's commonly used.

**Comment by andyferris** [comment]
No, not at this stage, sorry (there's no concept of time or dynamic datums in this package, yet).
